packages/twenty-server/src/engine/core-modules/user-workspace/user-workspace.entity.ts

```diff
@@ -58,4 +60,15 @@ export class UserWorkspace {
   @Field({ nullable: true })
   @Column({ nullable: true, type: 'timestamptz' })
   deletedAt: Date;
+
+  @Field({ nullable: false })
+  @Column({ default: false })
+  twoFactorEnabled: boolean;
+
+  @Field(() => [TwoFactorMethod], { nullable: true })
+  @OneToMany(
+    () => TwoFactorMethod,
+    (twoFactorMethod) => twoFactorMethod.userWorkspace,
+  )
+  twoFactorMethods: Relation<TwoFactorMethod[]>;
```

Comment:
I added the `twoFactorMethods` relation in `UserWorkspace`, but for some reason, typeorm can't seem to find the association

```ts
/Users/nicolasrouanne/dev/twenty/src/metadata-builder/EntityMetadataBuilder.ts:1111
        entityMetadata.relations.forEach((relation) => {
                                 ^
TypeORMError: Entity metadata for UserWorkspace#twoFactorMethods was not found. Check if you specified a correct entity object and if it's connected in the connection options.
    at /Users/nicolasrouanne/dev/twenty/src/metadata-builder/EntityMetadataBuilder.ts:1121:23
```

Reply:
After a quick chat with @FelixMalfait on Discord, it's probably a module-wiring issue. We are going to add the `TwoFactorModule` to `AuthModule`, to ensure it's properly loaded

Reply:
Actually fixed by @FelixMalfait on 59f84baca3c39463cb9f96e13f81ad5698cfe597 :pra